packages/tooltip/src/Tooltip.tsx

```diff
@@ -398,7 +380,7 @@ function Tooltip({
                 },
                 className,
               )}
-              ref={setTooltipNode}
+              ref={tooltipRef}
```

Comment:
Are there any downsides to removing the callback?

Reply:
Not that I see, no. That old way of doing refs is a bit of an anti-pattern afaik